src/http/fetch.rs

```diff
@@ -117,6 +157,8 @@ fn get_fetch_options<'js>(
     let mut method = None;
     let mut body = None;
     let mut abort_receiver = None;
+    let mut redirect = None;
```

Comment:
Maybe an empty mut string instead and then `push_str() if option is set`. Then we can avoid `unwrap_or_default.to_string()`?

Reply:
We have corrected the problem, please confirm.